packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/addLayerToFunctionWalkthrough.ts

```diff
@@ -38,6 +38,8 @@ export const addLayersToFunctionWalkthrough = async (
 
   if (lambdaLayers.length === 0) {
     context.print.info('No Lambda layers were selected');
+    const removeMessage = `Removing ${previousSelections.length} previously added Lambda layer`;
+    context.print.info(removeMessage + (previousSelections.length === 1 ? '' : 's') + ' from Lambda function');
```

Comment:
Since this flow is used for add and update, I would put this inside an `if (previousSelections.length > 0)` block to prevent it from being printed on add. Also nit I think it's a little cleaner to construct this message as:
```suggestion
    const plural = previousSelections.length > 1 ? 's' : '';
    const removeMessage = `Removing ${previousSelections.length} previously added Lambda layer${plural} from Lambda function`;
    context.print.info(removeMessage);
```